### ISSUE TITLE
New framework test not running

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#139156]
I added a new test in https://github.com/flutter/flutter/commit/e49cc81222cfc837e5066e3e9963c8ec7b382224, but it has not started running yet.

cc @sealesj

**Comment by sealesj** [comment]
@keyonghan Do you know why this new test would not be found in the "available builder list" log for this PR, but in all other PRs around this time I can see the new test in new available builders? 

![Image](https://github.com/flutter/flutter/assets/103135467/1ade3bc2-2957-496c-9c9e-7d3f47c55e9e)

**Comment by ricardoamador** [comment]
Wait. In the Current PR if the builder is new it would not be there until after it gets merged so this makes sense why it is not seen here in presubmit. @keyonghan would the builder be available in postsubmit though?

**Comment by keyonghan** [comment]
There was another instance of https://github.com/flutter/flutter/issues/122117 happened early this morning. The hanging should be due to `ci_yaml_rolelr` not rolled yet.

As of now, there is a yellow box without being schedule (shown below), I guess this was done by a manual run? Since the ci_yaml roller was still not rolled, it just hangs due to no builder available. Now the above incident was resolved and I just clicked another rerun, it is running now.
<img width="384" alt="Screenshot 2023-11-28 at 1 45 06 PM" src="https://github.com/flutter/flutter/assets/54558023/08819283-820a-4c17-a289-44b0bd9e3b34">
